### ISSUE TITLE
Add support for a `FOOTER` section

### ISSUE THREAD

**Issue by sschuberth** [jmongard/Git.SemVersioning.Gradle#41]
Similar to `HEADER`, support for a free-text `FOOTER` (which is empty by default) would be nice, e.g. to add optional notes to a release.

**Comment by jmongard** [comment]
Did you imagine a simple string that is appended last to the change log or something more elaborate?

https://github.com/jmongard/Git.SemVersioning.Gradle/commit/0e420b64efda595bcfefdd3a8c4e4525ca67adda

**Comment by sschuberth** [comment]
> Did you imagine a simple string that is appended last to the change log or something more elaborate?

A simple string should be enough!

**Comment by jmongard** [comment]
Footer is now available in v0.1.0

```kotlin
   changeLogTexts {
                    footer = "A footer"
   }
 ```
